src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -209,7 +208,21 @@ object BlockchainConfig {
 
       override val chainId: Byte = Hex.decode(blockchainConfig.getString("chain-id")).head
 
-      override val blockReward: UInt256 = UInt256(BigInt(blockchainConfig.getString("block-reward")))
+      override val monetaryPolicyConfig = MonetaryPolicyConfig(blockchainConfig.getConfig("monetary-policy"))
     }
   }
 }
+
+case class MonetaryPolicyConfig(
+  eraDuration: Int,
+  rewardRedutionRate: Double,
+  firstEraBlockReward: BigInt
+)
+
+object MonetaryPolicyConfig {
+  def apply(mpConfig: TypesafeConfig): MonetaryPolicyConfig = MonetaryPolicyConfig(
+    mpConfig.getInt("era-duration"),
+    mpConfig.getDouble("reward-reduction-rate"),
```

Comment:
I know it might be a strange/stupid case but If `reward-reduction-rate` is bigger than 1, then the `calculator.calcBlockMinerReward` returns a negative number which makes no sense. Should it be set as something like `Max(1, mpConfig.getDouble("reward-reduction-rate"))` ?